pkg/providers/instancetype/types.go

```diff
@@ -46,6 +46,9 @@ const (
 
 var (
 	instanceTypeScheme = regexp.MustCompile(`(^[a-z]+)(\-[0-9]+tb)?([0-9]+).*\.`)
+
+	instanceTypesSupportedForWindows = []string{"m4.16xlarge"}
```

Comment:
optional: scope these values to the `isSupportedForWindows` function since they are only used there